src/Nordea/Components/MultiSelectDropdown.elm

```diff
@@ -160,37 +178,48 @@ view attrs dropdown =
                         , maxHeight (rem 16.75)
                         , dropdownStyles
                         , listStyle none
+                        , Css.property "margin-block-start" "0"
+                        , Css.property "margin-block-end" "0"
+                        , Css.property "padding-inline-start" "0"
                         ]
                     ]
                     (dropdown.options |> List.map viewOption)
                 ]
     in
     Html.fieldset
         ([ Events.on "focusout" (Decode.succeed (dropdown.onFocus False))
-         , Events.on "focusin" (Decode.succeed (dropdown.onFocus True))
+         , Events.onClick (dropdown.onFocus (not dropdown.hasFocus))
          , tabindex 0
-         , css [ displayFlex, flexDirection column ]
+         , css
+            [ minWidth fitContent
+            , zIndex (int 1)
+            , displayFlex
+            , flexDirection column
+            , marginInlineStart (rem 0)
+            , marginInlineEnd (rem 0)
+            , Css.property "padding-block-start" "0"
+            , Css.property "padding-block-end" "0"
+            , Css.property "padding-inline-start" "0"
+            , Css.property "padding-inline-end" "0"
+            , Css.property "border" "none"
+            , Css.property "min-inline-size" "min-content"
+            ]
          ]
             ++ attrs
         )
         [ Html.row [ css [ displayFlex, justifyContent spaceBetween, marginBottom (rem 0.2) ] ]
             [ Label.init { label = dropdown.label } |> Label.withAsLegend |> Label.view []
             , dropdown.requirednessHint |> Html.viewMaybe RequirednessHint.view
             ]
-        , Html.div
-            [ class "nordea-select"
-            , css [ position relative ]
-            ]
+            |> Html.showIf (isLabel || isRequirednessHint)
+        , Html.div [ css [ position relative ] ]
```

Comment:
Do we need this div, or can the css be moved up to the fieldset and this div deleted?

Reply:
Deleted! Nice suggestion🙌